app/src/main/java/com/kickstarter/ui/viewholders/ActivitySampleFriendBackingViewHolder.kt

```diff
@@ -66,6 +70,11 @@ class ActivitySampleFriendBackingViewHolder(
         delegate.activitySampleFriendBackingViewHolderProjectClicked(this, activity.project())
     }
 
+    override fun destroy() {
+        this.vm.clearDisposables()
```

Comment:
Don't forget to clear the disposable in this viewholder too, not just the one in the viewmodel!

Reply:
done!